index.js

```diff
@@ -1,60 +1,56 @@
+module.exports = ApplicationConfig
+
 var fs = require('fs')
 var path = require('path')
 var mkdirp = require('mkdirp')
 var applicationConfigPath = require('application-config-path')
 
-function applicationConfig (name) {
-  var directoryPath = applicationConfigPath(name)
-  var filePath = path.join(directoryPath, 'config.json')
+function ApplicationConfig (name) {
+  if (!(this instanceof ApplicationConfig)) return new ApplicationConfig(name)
```

Comment:
There is something that rubs me the wrong way about this line since it will break when switching to the `class ApplicationConfig {}` syntax.

Do you have any input on this? What do you think of having module.exports be:

```javascript
module.exports = function createApplicationConfig (name) {
  return new ApplicationConfig(name)
}
```

I'm not sure exactly how I feel about this so input is very welcome 👍